torch/nn/modules/pooling.py

```diff
@@ -227,29 +226,29 @@ class MaxUnpool1d(_MaxUnpoolNd):
     including the indices of the maximal values and computes a partial inverse
     in which all non-maximal values are set to zero.
 
-    .. note:: `MaxPool1d` can map several input sizes to the same output sizes.
-              Hence, the inversion process can get ambiguous.
+    .. note:: :class:`MaxPool1d` can map several input sizes to the same output
```

Comment:
`~torch.nn.MaxPool1d`

Reply:
I don't think this is a problem. We did all the tensor.* docs this way. Moreover, the name will be clickable to redirect to MaxPool1d.